Concept/content/Screen01.ui.qml

```diff
@@ -1,71 +1,148 @@
+        Item {
+            id: spacerTitleHelp
+            anchors.left: conceptText.right
+            width: 800
+        }
+
+        // "Help" button
+        Button {
+            id: helpButton
+            text: "Help"
+            font.pixelSize: 20
+            anchors.left: spacerTitleHelp.right
+            anchors.verticalCenter: parent.verticalCenter
+        }
 
-    Button {
-        id: button
-        text: qsTr("Press me")
-        anchors.verticalCenter: parent.verticalCenter
-        checkable: true
-        anchors.horizontalCenter: parent.horizontalCenter
+        //spacing item
+        Item {
+            id: spacerHelpSearch
+            anchors.left: helpButton.right
+            width: 100
+        }
 
-        Connections {
-            target: button
-            onClicked: animation.start()
+        // Search Bar
+        TextField {
+            id: searchBar
+            placeholderText: "Search..."
+            width: 300
+            anchors.left: spacerHelpSearch.right
+            anchors.verticalCenter: parent.verticalCenter
         }
+
+        //spacing item
+        Item {
+            id: spacerSearchButton
+            anchors.left: searchBar.right
+            width: 40
+        }
+
+        // "Search" button
+        Button {
+            id: searchButton
+            text: "Search"
+            font.pixelSize: 20
+            anchors.left: spacerSearchButton.right
+            anchors.verticalCenter: parent.verticalCenter
+        }
+
+        //spacing item
+        Item {
+            width: 200
+            anchors.left: searchButton.right
+        }
+
     }
 
-    Text {
-        id: label
-        text: qsTr("Hello Concept")
-        anchors.top: button.bottom
-        font.family: Constants.font.family
-        anchors.topMargin: 45
-        anchors.horizontalCenter: parent.horizontalCenter
-
-        SequentialAnimation {
-            id: animation
-
-            ColorAnimation {
-                id: colorAnimation1
-                target: rectangle
-                property: "color"
-                to: "#2294c6"
-                from: Constants.backgroundColor
-            }
 
-            ColorAnimation {
-                id: colorAnimation2
-                target: rectangle
-                property: "color"
-                to: Constants.backgroundColor
-                from: "#2294c6"
+    // Main Content
+    Rectangle {
+        id: rectangle
+        width: parent.width
+        height: parent.height - nameBar.height
+
+        anchors.top: nameBar.bottom
+
+        color: Constants.backgroundColor
+
+        Button {
+            id: button
+            text: qsTr("Press me")
+            anchors.verticalCenter: parent.verticalCenter
+            checkable: true
+            anchors.horizontalCenter: parent.horizontalCenter
+
+            Connections {
+                target: button
+                onClicked: animation.start()
             }
         }
-    }
-    states: [

```

Comment:
states variables for the button become useless, but we might need it later for other state animations